DB/StoredProcedures/DeleteRevisionById.sql

```diff
@@ -0,0 +1,6 @@
+CREATE OR ALTER PROCEDURE usp_DeleteRevisionById @Id BIGINT AS 
+BEGIN
+    Delete
+    FROM dbo.Revisions
+    Where Id=@Id
```

Comment:
Also bind to Client and Car to prevent misuse.

Reply:
done - but will implement soft delete here